utils/field.h

```diff
@@ -30,6 +30,16 @@ enum display_field_id {
 	GRAPH_F_TASK_TOTAL_TIME = 0,
 	GRAPH_F_TASK_SELF_TIME,
 	GRAPH_F_TASK_TID,
+
+	REPORT_F_TOTAL_TIME      = 0,
+	REPORT_F_TOTAL_TIMEAVG,
+	REPORT_F_TOTAL_TIMEMIN,
+	REPORT_F_TOTAL_TIMEMAX,
```

Comment:
It would be nice to change as follows.
```suggestion
	REPORT_F_TOTAL_TIME_AVG,
	REPORT_F_TOTAL_TIME_MIN,
	REPORT_F_TOTAL_TIME_MAX,
```

Reply:
Ok. thank you.